src/version.cpp

```diff
@@ -1,15 +1,5 @@
-#if (defined _WIN32 || defined WINDOWS || defined MINGW) && ! defined CROSS_LINUX && !defined _MSC_VER
-
-#ifndef VERSION
-#define VERSION "0.C" // FIXME: automatically generate VERSION based on git
```

Comment:
Possibly CMake and Make builds should inject a define to control importing version.h instead of us using build system matching.